plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginAutoInstallTest.kt

```diff
@@ -162,6 +167,49 @@ class SentryPluginAutoInstallTest(
         assertFalse { "FAILED" in result.output }
     }
 
+    @Test
+    fun `sqlite is not added for lower sentry versions`() {
+        appBuildFile.appendText(
+            // language=Groovy
+            """
+            dependencies {
+              implementation 'androidx.sqlite:sqlite:2.0.0'
+            }
+
+            sentry.autoInstallation.enabled = true
+            sentry.autoInstallation.sentryVersion = "6.20.0"
```

Comment:
`l`: I guess we can be in an unlucky situation where the gradle plugin has been updated to the latest version, but the sentry version is not 6.21.0, so our users will lose on the database instrumentation. But this is probably a documentation thing, and maybe we should also mention in the changelog and migration guide.

Reply:
actually, now when I'm thinking about it - maybe we shouldn't wipe the old implementation completely, but rather keep it for compatibility? We could have two checks `isNewDatabaseInstEnabled` and `isOldDatabaseInstEnabled` - the first one would check for the `sentry-android-sqlite` presence like it does now, and if it's not available, we'd fallback to the old instrumentation (in case it's enabled) - how does that sound? 

We can think of removing it completely for the next major, for example.

Reply:
yeah, it makes sense